sdk/monitor/ci.yml

```diff
@@ -24,6 +24,7 @@ extends:
   parameters:
     ServiceDirectory: monitor
     RunUnitTests: true
+    TestProxy: true
```

Comment:
Recently Harsha updated the pipelines so that this variable defaults to true, so this isn't needed anymore :)

```suggestion
```